### PR TITLE
Quelques petits ajouts histoire de se simplifier la vie

### PR DESCRIPTION
**Pull request by LotuxPunk**

### CODE REVIEW

File: da/scripts/PseudoCodeParser.js [hunk 1]
```diff
@@ -21,7 +21,9 @@ var PseudoCodeParser = function(ownValues) {
         { pattern: /<->/g, replacement: "←→" },
         { pattern: /->/g, replacement: "→" },
         { pattern: /=>/g, replacement: "⇒" },
-        { pattern: /sqrt\^/g, replacement: "√" }
+        { pattern: /sqrt\^/g, replacement: "√" },
+        { pattern: /&&/g, replacement: "AND" }
+
```
Comment (gaetanledocte) [inline da/scripts/PseudoCodeParser.js]: Remove whitespace here

File: da/index.html [hunk 2]
```diff
@@ -34,7 +34,8 @@ <h2 class="welcome-subtitle">Parseur de pseudo-code</h2>
     <header class="topbar">
         <h1 class="topbar-title">Editeur de diagrammes d'actions (pseudo-code)</h1>
         <nav class="topbar-menu">
-            <a href="#" class="menu-item" id="buttonNew">Nouveau</a>
+			<a href="#" class="menu-item" id="buttonNew">Nouveau</a>
+			<a href="#" class="menu-item" id="buttonSaveInCache">Sauvegarder</a>
```
Comment (gaetanledocte) [inline da/index.html]: Pay attention to the indentation please :)

File: da/scripts/app.js [hunk 3]
```diff
@@ -26,6 +26,14 @@ window.addEventListener("load", function() {
     drawDiagram();
 });
 
+//CTRL + S
+document.addEventListener("keydown", function(event) {
+    if ((window.navigator.platform.match("Mac") ? event.metaKey : event.ctrlKey)  && event.keyCode == 83) {
+      event.preventDefault();
+      saveContentToCache();
+    }
+  }, false);
```
Comment (gaetanledocte) [inline da/scripts/app.js]: Pay attention to the indentation please :)

**Review by gaetanledocte — CHANGES_REQUESTED** [review]
- Review indentation
- Remove useless whitespaces

File: da/index.html (hunk 2 above)
Comment (LotuxPunk) [reply da/index.html]: image
This is embarrassing. On VisualCode, there are no intentation problems.

**Comment by LotuxPunk** [comment]
Normally, everything is good now.

**Review by gaetanledocte — APPROVED** [review]

**Pull request merged**